src/OVAL/oval_sexp.c

```diff
@@ -719,6 +720,12 @@ int oval_state_to_sexp(void *sess, struct oval_state *state, SEXP_t **out_sexp)
 			SEXP_free(r0);
 		}
 
+		oext = oval_state_content_get_check_existence(content);
+		if (oext != OVAL_EXISTENCE_UNKNOWN) {
```

Comment:
I this case, the block after the condition will not be executed, that means the check_existence attribute will not affect the probe at all.